src/core/service.c

```diff
@@ -5229,38 +5229,21 @@ static int service_live_mount(Unit *u,
 
         r = service_arm_timer(s, /* relative= */ true, s->timeout_start_usec);
         if (r < 0) {
-                log_unit_warning_errno(u, r, "Failed to install timer: %m");
-                sd_bus_error_set_errnof(
-                                error,
-                                r,
-                                "Live mounting '%s' on '%s' for unit '%s' cannot be scheduled: failed to install timer",
-                                src,
-                                dst,
-                                u->id);
+                log_unit_error_errno(u, r, "Failed to install timer: %m");
```

Comment:
I do not think it is necessary to raise the log level. Hopefully the service manager can continue to work even this is failed.

Reply:
We certainly would continue, but a basic operation fails which is worth `error` level logging I'd say.